vite-plugin-ruby/src/index.ts

```diff
@@ -37,11 +37,11 @@ function config (userConfig: UserConfig, env: ConfigEnv): UserConfig {
   const fs = { allow: [projectRoot], strict: true }
   const server = { host, https, port, strictPort: true, fs }
 
-  const isProduction = config.mode === 'production'
+  const isDevelopment = config.mode === 'development'
```

Comment:
For the purpose of removing old assets in the local environment:

```suggestion
  const isDevelopment = config.mode === 'development' || config.mode === 'test'
```